.github/workflows/ts-packages.yml

```diff
@@ -60,6 +60,53 @@ jobs:
           ENABLE_GAS_REPORT: 1
       - name: Print gas report
         run: cat packages/contracts/gas-report.txt
+
+  test-coverage:
```

Comment:
Rather than place this under the above `test` job, I broke this out into a new job in order to parallelize. This does result in a fair bit of duplicate configuration, perhaps there's a better way?

Reply:
Unfortunately not :( agree that the duplication is not ideal